packages/python-sat/test_python_sat.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from pyodide_test_runner import run_in_pyodide
+
+solvers = [
+    "cadical",
+    "gluecard30",
+    "gluecard41",
+    "glucose30",
+    "glucose41",
+    "maplechrono",
+    "maplecm",
+    "maplesat",
+    "mergesat3",
+    "minicard",
+    "minisat22",
+    "minisat-gh",
+]
+
+
+@pytest.mark.parametrize("name", solvers)
+@run_in_pyodide(packages=["python-sat"])
+def test_solvers(selenium, name):
+
+    from pysat.formula import CNF
+    from pysat.solvers import Solver
+
+    cnf = CNF(from_clauses=[[1, 2, 3], [-1, 2], [-2]])
+
+    with Solver(name=name, bootstrap_with=cnf) as solver:
+        solver.solve()
+        stats = solver.accum_stats()
+        assert "conflicts" in stats, f"No conflicts for {name}"
+        # assert 'decisions' in stats, 'No decisions for {0}'.format(name)
+        # assert 'propagations' in stats, 'No propagations for {0}'.format(name)
```

Comment:
@pysathq @alexeyignatiev why do these assertions fail?

Reply:
@hoodmane, I believe this is because the first argument of `CardEnc.atmost()` should be called `lits` instead of `literals`, see line 75.